requirements.txt

```
@@ -18,6 +18,9 @@ datasketch>=1.5.6,<1.6.0
 google==3.0.0  # safesearch filtering
 more_itertools>=8.12.0,<8.14.0
 pandas>=1.0.0,<2.0.0
+emoji~=1.7.0
+luga~=0.2.5
+langdetect~=1.0.9
```

```suggestion
emoji>=1.7.0,<1.8.0
luga>=0.2.5,<0.3.0
langdetect>=1.0.9,<1.1.0
```
As I understand it the `~=` operator allows for versions X.X.*, this allows for a lower and potentially bugged version.

---

The change you made covers the same versions, but I guess are more readable